api/version_test.go

```diff
@@ -0,0 +1,13 @@
+package api
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestLibwasmvmVersion(t *testing.T) {
+	version, err := LibwasmvmVersion()
+	require.NoError(t, err)
+	require.Equal(t, "1.0.0", version)
```

Comment:
Will this have to be updated with  every version change? Better to use a `VERSION` tag of ref from somewhere. Perhaps some dirty code to parse the `Cargo.toml` from Go? That, or do some structural checks here only.

Reply:
Right, this is a leftover from development state. Ideally we use a simple regex here to ensure we get something sane.

Reply:
Done in 7235e1b157e3cf843b4879006468197e04eb4416